rmgpy/rmg/main.py

```diff
@@ -898,6 +902,27 @@ def execute(self, **kwargs):
                 
                 plot_sensitivity(self.outputDirectory, index, reactionSystem.sensitiveSpecies)
 
+        # Check all core reactions (in both directions) for collision limit violation
+        logging.info("Checking core reactions for collision rate limit violators...")
+        violators = []
+        for rxn in self.reactionModel.core.reactions:
+            f_violator, r_violator = rxn.check_collision_limit_violation(t_min=self.Tmin, t_max=self.Tmax,
+                                                            p_min=self.Pmin, p_max=self.Pmax)
+            if f_violator is not None:
```

Comment:
I think this statement can be simplified to `if f_violator:`

Reply:
`if x` and `if x is not None` are used in different contexts. Check the PEP 8 recommendations [here](https://legacy.python.org/dev/peps/pep-0008/#programming-recommendations).

Reply:
Thanks @cgrambow, it was helpful reading this piece. I'll leave the `is not None` statement as as